puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
If the process for a new project is `composer create-project` and `vagrant up` I this is is a huge inconvenience. Even if you knew you wanted to delete lines the process becomes `composer create-project`, scan though lines of code deleting things you _may_ not need afraid to remove too much in case you break the puppet build, then `vagrant up` and hope you didn't remove something like intl and have to add it back and re-provision the box. This isn't event worst case scenario. I could see someone making an initial commit with all of this in the repo only to have to make another commit down the line to remove it all and re-buld the box and have every other dev do the same only to add one or two things back when the client asks for a feature that requires gd or geoip. `composer create-project` and `vagrant up` seems much more sane as I'm much more likely to know "I need imagick" and think "I better check that it will be installed" versus "I don't need, g++, graphviz, make..." and think "I better make sure those are not going to be installed".